### PR TITLE
Feat/creator hub screen

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
Creator screen design made.

### CODE REVIEW

File: lib/screens/creator_hub/creator_hub_screen.dart [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class _CreatorHubScreenState extends State<CreatorHubScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        color: EaselAppTheme.kWhite,
+        child:  SafeArea(
+        child: Scaffold(
+          body:
+          Text("gfg"),
```
Comment (kjawadDeveloper2) [inline lib/screens/creator_hub/creator_hub_screen.dart]: Remove this.

File: lib/utils/route_util.dart [hunk 2]
```diff
@@ -4,5 +4,6 @@ class RouteUtil {
   static String ROUTE_TUTORIAL = "/tutorial";
   static String ROUTE_WELCOME = "/welcome";
   static String ROUTE_HOME = "/home";
+  static String ROUTE_CREATOR_HUB = "/home";
```
Comment (kjawadDeveloper2) [inline lib/utils/route_util.dart]: Why this is home?

File: ios/Podfile.lock [hunk 3]
```diff
@@ -1,14 +1,14 @@
 PODS:
```
Comment (kjawadDeveloper2) [inline ios/Podfile.lock]: Add this file in gitignore.

File: lib/screens/creator_hub/creator_hub_screen.dart [hunk 4]
```diff
@@ -10,14 +13,172 @@ class CreatorHubScreen extends StatefulWidget {
 }
 
 class _CreatorHubScreenState extends State<CreatorHubScreen> {
+  bool collapsed=true;
```
Comment (kjawadDeveloper2) [inline lib/screens/creator_hub/creator_hub_screen.dart]: Formatting issue.

File: lib/screens/creator_hub/creator_hub_screen.dart [hunk 5]
```diff
@@ -10,14 +13,172 @@ class CreatorHubScreen extends StatefulWidget {
...
+            ),
+          ),
         )));
   }
+
+  Widget animatedContainer({required String title}){
+   return AnimatedContainer(
+      duration: const Duration(milliseconds: 100),
+      height: collapsed ? 40.h : 150.h,
+      // width: 300.w,
```
Comment (kjawadDeveloper2) [inline lib/screens/creator_hub/creator_hub_screen.dart]: Remove this dead code.

File: lib/screens/creator_hub/creator_hub_screen.dart [hunk 6]
```diff
@@ -10,14 +13,172 @@ class CreatorHubScreen extends StatefulWidget {
...
+                          decoration: const BoxDecoration(border: Border(bottom: BorderSide(color: EaselAppTheme.kBlack, width: 2))),
+                        ),
+                        CustomPaint(size: Size(10.w, 10.h), painter: DiagonalLinePainter()),
+                      ],
+                    ),
+
+                    SizedBox(
+                      height: 10.h,
+                    ),
+                    //  if (!collapsed && widget.name == 'Ownership') ...listOnwership else if (!collapsed && widget.name == 'NFT Detail') ...listDetails
```
Comment (kjawadDeveloper2) [inline lib/screens/creator_hub/creator_hub_screen.dart]: Remove this dead code.

**Review by kjawadDeveloper2 — COMMENTED** [review]
Kindly solve these issues and also check on the tablet.

**Comment by kjawadDeveloper2** [comment]
Podfile.lock issue is not yet resolved.

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

**Review by kiroj305 — APPROVED** [review]
Looks like the creator hub has not been fully implemented in this pr, so assume this could be round 1. I approve if that's the case.

**Comment by kjawadDeveloper2** [comment]
@Tiny-Angel305  yes thats the case.

**Pull request merged**